guidelines/round-table.md

```diff
@@ -0,0 +1,49 @@
+## Data Together Weekly Roundtable Call
+
+Once a week on **Mondays 5-5:30pm**, a rotating facilitator will lead a round table discussion of a Data Together-related topic, challenge, or question. We use a standing [Zoom call link](https://edgi-video-call-landing-page.herokuapp.com/https://zoom.us/j/455833201): [zoom.us/j/455833201](https://edgi-video-call-landing-page.herokuapp.com/https://zoom.us/j/455833201) and [GitHub `roundtable` issues](https://github.com/datatogether/datatogether/labels/roundtable) to run each call. **Check out [how to host](#how-to-host)** before running your first call.
+
+### Objectives
```

Comment:
Your call, but maybe bolding operative bits of each bullet? ie. "ongoing work", "larger issues", "developing ideas", "maintain momentum"? You have made terse-skills dc, but thinking about the skimmers :)